File: URI/python/1012.py
A, B, C = [float(x) for x in input().split()]
pi = 3.14159

triangle = (A * C) / 2
circle = pi * C ** 2
trapezium = ((A + B) * C) / 2
square = B ** 2
rectangle = A * B

print('TRIANGULO: {:.3f}'.format(triangle))
print('CIRCULO: {:.3f}'.format(circle))
print('TRAPEZIO: {:.3f}'.format(trapezium))
print('QUADRADO: {:.3f}'.format(square))
print('RETANGULO: {:.3f}'.format(rectangle))

File: URI/C/1012.c
#include <stdio.h>
#define PI 3.14159

int main() {

    float A, B, C, area;
    
    scanf("%f %f %f", &A, &B, &C);

    printf("TRIANGULO: %.3f\n", (A * C)/2);
    printf("CIRCULO: %.3f\n", PI * C * C);
    printf("TRAPEZIO: %.3f\n", C * (A + B)/2);
    printf("QUADRADO: %.3f\n", B * B);
    printf("RETANGULO: %.3f\n", A * B);

    return 0;

}